fastlane/Fastfile

```diff
@@ -35,4 +35,15 @@ platform :ios do
     perform_release target: 'SimpleKeychain-iOS'
     publish_release repository: 'SimpleKeychain'
   end
+
+  desc "Generate API documentation"
+  lane :build_docs do
+    # Work from project root (defaults to /fastlane)
+    Dir.chdir("..") do
+      # Generate the .doccarchive
+      sh "xcodebuild docbuild -scheme SimpleKeychain-iOS -configuration Release -destination 'generic/platform=iOS' -derivedDataPath docs.archive"
+      # Generate the static site
+      sh "$(xcrun --find docc) process-archive transform-for-static-hosting docs.archive/Build/Products/Release-iphoneos/SimpleKeychain.doccarchive --hosting-base-path SimpleKeychain --output-path docs"
```

Comment:
You were right about `--hosting-base-path` being necessary 👍 I've already made the change over on JWTDecode.swift to reflect this as well.